frontend/lib/tests/form.test.tsx

```diff
@@ -38,16 +43,17 @@ describe('Form', () => {
         phoneNumber: simpleFormErrors('bar')
       }
     };
-    const form = mount(<MyForm onSubmit={jest.fn()} errors={errors} isLoading={false} />);
-    const html = form.html();
+    const pal = new ReactTestingLibraryPal(<MyForm onSubmit={jest.fn()} errors={errors} isLoading={false} />);
+    const html = pal.rr.container.innerHTML;
```

Comment:
I was tempted to add an accessor to `ReactTestingLibraryPal` that made it easier to get `rr.container.innerHTML` but I decided against it--examining the raw HTML of a component doesn't seem like a very granular or resilient way of making sense of a component's behavior and I don't think we should encourage it.